_episodes/06-includes.md

```diff
@@ -100,7 +125,7 @@ at the top of your page.
 > > ## Solution
 > >
 > > ```
-> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="carpentries-incubator.github.io/building-websites-with-jekyll-and-github-or-gitlab/files/site_banner.png"></a>
+> > <a href="https://YOUR_USERNAME.github.io/YOUR_REPO_NAME/"><img src="/images/site_banner.png" alt="Group Website with Jekyll"></a>
```

Comment:
👍 for alt text

Reply:
Maybe we should write why alt text is important?

Reply:
tried to address this in 4a2a87a